### ISSUE TITLE
kats require high version of numpy

### ISSUE THREAD

**Issue by Adair1990** [facebookresearch/Kats#290]
could kats require low version of numpy？ so it will not generate conflicts with other python packages. i want use kats in the pyspark platform but i conflict with the low version of pyarrow==0.14.0.thanks

**Comment by ujjwalbhatnagar** [comment]
Yes, Facebook Research KATS can require a low version of Numpy. The most recent version of KATS is tested with Numpy 1.19.x, which is a low version of Numpy. However, you will need to ensure that any other packages used with KATS are compatible with the same version of Numpy. Additionally, it is important to note that even with a low version of Numpy, there may still be conflicts with other Python packages.
